components/channel_header_dropdown/mobile_channel_header_dropdown.tsx

```diff
@@ -2,9 +2,12 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
 
+import {UserProfile} from '@mattermost/types/users.js';
+
+import {Channel} from '@mattermost/types/channels.js';
+
```

Comment:
```suggestion
import {UserProfile} from '@mattermost/types/users';
import {Channel} from '@mattermost/types/channels';

```
there are not `js` files